lib/combineReducers.lua

```diff
@@ -3,6 +3,11 @@
 ]]
 local function combineReducers(map)
 	return function(state, action)
+		-- If state is nil, substitute it with a blank table.
+		if state == nil then
+			state = {}
+		end
```

Comment:
Should we do more validation of the state table here or just assume that the user won't pass something dumb into `initialState`?

Reply:
I think we can safely assume they're not going to pass something weird to `initialState` if they're using `combineReducers` to begin with.